Documentation/gitlab-ci.md

```diff
@@ -175,6 +175,105 @@ check_interval = 0  # Default poll interval
     ]
 ```
 
+#### Mender's gitlab-master configuration reference
+
+The content of `/etc/gitlab-runner/config.toml` at the time of writing follow:
```

Comment:
nit: suggested change: "The content ... _follows_", or "The _contents_ ... follow".